src/domain/event/utils.ts

```diff
@@ -169,3 +170,20 @@ export const getEnrolmentType = (
   }
   return EnrolmentType.Unenrollable;
 };
+
+/**
+ * Does the event support enrolling in the queue?
+ * 1. Are the enrolments handled internally in the Kultus
+ * 2. Has the enrolment started already
+ * 3. Are there any occurrences for the event
+ */
```

Comment:
How would you feel about this:
```typescript
/**
 * Does the event support enrolling in the queue?
 * @return True if enrolments are handled internally in Kultus,
 * the enrolment has started and there is at least one occurrence
 * for the event, otherwise False.
 */
```

Reply:
Quite trivial thing. I think that a list is more clear, but I can still change this, just for the `@return ` jsdoc syntax 😄

Reply:
To myself the list was otherwise clear but whether the bullet points meant OR or AND was not clear and that's why I suggested and alternative.